src/components/sync-sites-modal-selector.tsx

```diff
@@ -302,7 +333,8 @@ function SiteItem( {
 							getIpcApi().openURL( `https://wordpress.com/hosting-features/${ site.id }` )
 						}
 					>
-						{ __( 'Enable hosting features ↗' ) }
+						{ __( 'Enable hosting features' ) }
```

Comment:
Is it intentional?

Reply:
Yes, the [same as above](https://github.com/Automattic/studio/pull/1406#discussion_r2093073689).